src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -264,7 +264,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     /**
      * @param array<string, string> $uses
      */
-    private function fixFunction(FunctionsAnalyzer $functionsAnalyzer, Tokens $tokens, int $index, array $uses, string $namespaceName): void
+    private function fixFunction(FunctionsAnalyzer $functionsAnalyzer, Tokens $tokens, int $index, array &$uses, string $namespaceName): void
```

Comment:
non-blocking

I took a look how this `uses` is used and passed around most of the methods.
maybe it could be state property and avoid pass-by-reference in all the methods 😅

Reply:
If anything, I would prefer creating some object representing a list of imports, so it's always passed by reference. Explicit arguments are better than quasi-global state.